charts/replicated-library/values-example.yaml

```diff
@@ -364,3 +372,82 @@ persistence:
           requests:
             storage: 8Gi
         storageClassName: slow
+
+
+# -- Configure the serviceAccounts for the chart here.
+# ServiceAccounts can be added by adding a dictionary key similar to the 'example' serviceAccount.
+# By default the name of the serviceAccount will be the name of the dictionary key unless overridden with serviceAccounts.*.nameOverride
+# @default -- See below
+serviceAccounts:
+  example:
+    # -- Enables or disables the service account
+    enabled: false
+    # -- Labels to add to the service account
+    labels: {}
+    # -- Annotations to add to the service account
+    annotations: {}
+    # -- Override the name of this object. Default name if not overwritten will be releaseName-ChartName-objectName
+    fullNameOverride:
+
+# -- Configure the roles for the chart here.
+# Roles can be added by adding a dictionary key similar to the 'example' role.
+# By default the name of the role will be the name of the dictionary key unless overridden with roles.*.nameOverride
+# TODO: implement aggregated ClusterRoles
+# @default -- See below
+roles:
+  example:
+    # -- Enables or disables the role
+    enabled: false
+    # -- Labels to add to the role
+    labels: {}
+    # -- Annotations to add to the role
+    annotations: {}
+    # -- Override the name of this object. Default name if not overwritten will be releaseName-ChartName-objectName
+    fullNameOverride:
+    # -- Type of role. Must be either: ["Role", "ClusterRole"]
+    kind: Role
+    # -- Define the selectors used for aggregated ClusterRoles
+    aggregationRule: {
```

Comment:
This looks like the lone difference between Roles and ClusterRoles. Perhaps we should make it a bit more known that this is for one and not the other? Also I don't think this is being used in the `Role` template?

Reply:
@diamonwiggins can do, and no it's not used in the Role type

Reply:
I have it as a to-do to implement it for ClusterRoles but I can add some validation

Reply:
OK, I just went ahead and added the aggregate ClusterRole template